modules/programs/sftpman.nix

```
@@ -0,0 +1,101 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.sftpman;
+
+  jsonFormat = pkgs.formats.json { };
+
+  mountOpts = { config, name, ... }: {
+    options = {
+      host = mkOption {
+        type = types.str;
+        description = "The host to connect to.";
+        default = null;
+      };
+      port = mkOption {
```

Please separate options by an empty line:
```suggestion

      port = mkOption {
```